docs/guides/extensions/panels-display-data-vonage.md

```diff
@@ -0,0 +1,769 @@
+---
+description: A guide on how you can create a panel to display data from an external API like Vonage.
+contributors: Tim Butterfield, Kevin Lewis
+---
+
+# Create A Panel To Display External API Data With Vonage
+
+Panels are used in dashboards as part of the Insights module, and typically allow users to better-understand data held
+in their Directus collections. In this guide, you will instead fetch data from an external API and display it in a table
+as part of a panel.
+
+<img src="https://marketing.directus.app/assets/31ac7437-99ed-44fe-9b75-b21c92198fda.png" alt="Table with header Messages shows several items with status, sent delative date, a recipient ID, and a provider">
+
+Panels can only talk to internal Directus services, and can't reliably make external web requests. To create a panel
```

Comment:
Lets add _why_. It's because the browser security protections prevent those cross-origin requests from being made